pynucastro/nucdata/halflife_table.py

```diff
@@ -0,0 +1,51 @@
+import os
+
+
+class HalfLifeTable:
+    """
+    Read the table of halflives (in seconds) extracted from the Nubase
+    compilation in halflife2020.txt
+    """
+
+    def __init__(self, filename=None):
+
+        self.halflife = {}
+
+        if filename:
+            self.filename = filename
+        else:
+            datafile_name = 'halflife2020.txt'
+            nucdata_dir = os.path.dirname(os.path.realpath(__file__))
+            self.filename = os.path.join(os.path.join(nucdata_dir, 'AtomicMassEvaluation'), datafile_name)
```

Comment:
`os.path.join()` can take multiple path segments:
```suggestion
            self.filename = os.path.join(nucdata_dir, 'AtomicMassEvaluation', datafile_name)
```